docs/changelog.mdx

```diff
@@ -4,6 +4,24 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.17
+
+_February 29, 2024_
+
+**New Features**
+- Introducing an [automated UI testing framework for Unity](/automated-testing/overview)
+- An entirely new way of [representating states](/integrating-with-unity/defining-states) requiring less custom integration
+- [Rolling Out] New UI design - send us a message for access!
+
+
+**Bug Fixes**
+- The SDK is now available within production builds (vs just the Unity Editor)
```

Comment:
.. sort of

I would say something like

The SDK is still only usable in the editor, but will no longer interfere with creating production builds.  Future support for running the SDK in production builds will be coming at a later date.

Reply:
Ah good clarification, thanks!